### PR TITLE
[issue: 9] Converter should put converted files into "dot" folder

### PR DESCRIPTION
**Pull request by nitind**
Opted for a folder named "local_dashboards" so it remains visible
    in the tree, but still contains all the generated local dashboards.

(c) Copyright IBM Corp. 2015

### CODE REVIEW

**Comment by nitind** [comment]
For #9 .

**Comment by parente** [comment]
Travis failed on npm. I restarted it by clicking through and clicking the loopy-arrow restart build in the Travis build details.

File: urth/dashboard/nbexts/bundle_handler.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def _get_local_app(self, abs_nb_path):
 
         :param abs_nb_path:
         '''
-        md = self._create_app_bundle(abs_nb_path, bundle_root=self.nb_dir, overwrite=True, template_fn='local.tpl')
+        md = self._create_app_bundle(abs_nb_path, bundle_root=os.path.join(self.nb_dir, 'local_dashboards'), overwrite=True, template_fn='local.tpl')
```
Comment (parente) [inline urth/dashboard/nbexts/bundle_handler.py]: I'm guessing / hoping the code down here makes the directory if it does not exist?

**Comment by nitind** [comment]
Yes, bundle_handler.py's _get_local_app() calls _create_app_bundle() which uses os.makedirs() before calling the converter.

File: urth/dashboard/nbexts/bundle_handler.py (hunk 1 above)
Comment (nitind) [inline urth/dashboard/nbexts/bundle_handler.py]: Yes, bundle_handler.py's _get_local_app() calls _create_app_bundle() which uses os.makedirs() before calling the converter.

**Pull request merged**